internal/db/sentinel.go

```diff
@@ -0,0 +1,36 @@
+package db
```

Comment:
Can we move this to a `sentinel` package? I think `sentinel.Prefix()` reads better than `db.Prefix()` (likewise for the other functions)